### DISCUSSION TITLE
Splitting and Grouping Desktops

### DISCUSSION THREAD

**Discussion by xXGoziXx** [rocksdanister/lively#685] [Ideas]
Would it be possible to add muti monitor support for grouping or splitting monitors. Some of us have screens that we don't want the wallpaper running on.

**Comment by rocksdanister** [comment]
I don't understand what you mean, example would help.

**Reply by xXGoziXx** [reply]
Ok, for example I have 3 displays. I have 2 screens on my laptop (ROG Zephyrus Duo). I would like to group them together as 1 display, have a wallpaper span them and then have a different wallpaper on the 3rd one. 
![image](https://user-images.githubusercontent.com/9575063/131318769-f8e4af91-1f09-4f61-9b17-1436111cb635.png)
Other use cases may involve having 2 monitors and a laptop screen e.g. wanting to span the wallpaper on the monitors which are similar in size and have a separate wallpaper for the smaller laptop screen.
[Wallpaper Engine](https://www.wallpaperengine.io/en) has this feature but I prefer some of the wallpapers on this application more and would love to switch to this.

Before Grouping:
![image](https://user-images.githubusercontent.com/9575063/131319102-4fc7b93d-7567-4046-ac03-169c2115b6ee.png)
After Grouping:
![image](https://user-images.githubusercontent.com/9575063/131319213-b8fab18d-4933-4053-83c2-2957792b9b09.png)

Hope this helps! :)

**Reply by xXGoziXx** [reply]
I know it's probably hard to implement but I honestly wouldn't mind paying to have this feature. Maybe you can make it a premium/in-app purchase ^ ^

**Reply by hayden-flintoft** [reply]
This is one feature I use in wallpaper engine. I have a laptop and two identical monitors. I would like to span the wallpaper across the monitors and have a separate wallpaper for the laptop as per the use case above.

**Comment by rocksdanister** [comment]
Made issue tracker https://github.com/rocksdanister/lively/issues/1396
